docs/rules/no-multiple-empty-lines.md

```diff
@@ -54,16 +54,43 @@ var bar = 3;
 
 ```
 
-Examples of **correct** code for this rule with the `{ max: 2, maxEOF: 1 }` options:
+Examples of **correct** code for this rule with the `{ max: 2, maxEOF: 0 }` options:
 
 ```js
-/*eslint no-multiple-empty-lines: ["error", { "max": 2, "maxEOF": 1 }]*/
+/*eslint no-multiple-empty-lines: ["error", { "max": 2, "maxEOF": 0 }]*/
 
 var foo = 5;
 
 
 var bar = 3;
+```
+
+**Note**: Although this ensures zero empty lines at the EOF, most editors will still show one empty line at the end, as illustrated below. There is no empty line at the end of a file after the last `\n`, although editors may show an additional line. A true additional line would be represented by `\n\n`.
```

Comment:
Looks good, just a small note about "most editors will still show one empty line at the end": I think that this happens only if the file ends with a linebreak, so maybe we should add that to this sentence?

Reply:
I've added that to the sentence, I believe it is clear enough now. Let me now.